cql/evaluate-measure-hapi.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# Usage: ./evaluate-measure.sh -f <query>.cql <server-base>
+
+# Takes a CQL file, creates a Library resource from it, references that from a
+# Measure resource and calls $evaluate-measure on it.
+
+library() {
+cat <<END
+{
+  "resourceType": "Library",
+  "status": "active",
+  "type" : {
+    "coding" : [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/library-type",
+        "code" : "logic-library"
+      }
+    ]
+  },
+  "content": [
+    {
+      "contentType": "text/cql"
+    }
+  ]
+}
+END
+}
+
+measure() {
+cat <<END
+{
+  "resourceType": "Measure",
+  "status": "active",
+  "subjectCodeableConcept": {
+    "coding": [
+      {
+        "system": "http://hl7.org/fhir/resource-types",
+        "code": "Patient"
+      }
+    ]
+  },
+  "scoring": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/measure-scoring",
+        "code": "cohort"
+      }
+    ]
+  },
+  "group": [
+    {
+      "population": [
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "initial-population"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/cql",
+            "expression": "InInitialPopulation"
+          }
+        }
+      ]
+    }
+  ]
+}
+END
+}
+
+create-library() {
+  library | jq -cM ".url = \"urn:uuid:$1\" | .name = \"$1\" | .content[0].data = \"$2\""
```

Comment:
Blaze also will work with the added name. So we could use just one script for both with this added name.